airflow/api_fastapi/core_api/routes/public/task_instances.py

```diff
@@ -603,7 +622,10 @@ def get_mapped_task_instance_try_details(
 @task_instances_router.post(
     "/clearTaskInstances",
     responses=create_openapi_http_exception_doc([status.HTTP_404_NOT_FOUND]),
-    dependencies=[Depends(action_logging())],
+    dependencies=[
+        Depends(action_logging()),
+        Depends(requires_access_dag(method="PUT", access_entity=DagAccessEntity.TASK_INSTANCE)),
```

Comment:
```suggestion
        Depends(requires_access_dag(method="POST", access_entity=DagAccessEntity.TASK_INSTANCE)),
```

should it be post instead?

Reply:
used PUT as we are mutating. refer to https://github.com/apache/airflow/pull/47532#discussion_r1988827346

Reply:
got it. Thanks for clarification!